charts/flink-job/templates/podmonitor.yaml

```diff
@@ -27,4 +27,7 @@ spec:
       relabelings:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+  {{- if .Values.podMonitor.sampleLimit }}
```

Comment:
Use the `with` syntax whenever you can - it's cleaner and less verbose.

Reply:
I used `with` but was suggested to use `if`. cc: @sabw8217